sale_order_packaging_info/models/res_company.py

```diff
@@ -0,0 +1,15 @@
+# SPDX-FileCopyrightText: 2023 Coop IT Easy SC
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    packaging_product_ids = fields.Many2many(
+        comodel_name="product.product",
+        relation="res_company_product_product_packaging_rel",
```

Comment:
why does the `relation` needs to be explicitly set instead of using the default value?

Reply:
I'm fearful that it creates/uses `res_company_product_product_rel`, which may also be used by a different module that creates a M2m relationship between these two models.

Maybe I'm misunderstanding the default behaviour.

Reply:
good point! [the default](https://github.com/coopiteasy/OCB/blob/12.0/odoo/fields.py#L2689) uses indeed only the table names. a comment would be useful then.

Reply:
Comment added.